components/site/docsNavigation.tsx

```diff
@@ -1,5 +1,242 @@
+import React from 'react';
+import { getWindow } from 'utils/ssr';
+import Link from 'next/link';
+import { Box, BoxProps } from '@sparkpost/matchbox';
+import { PeopleOutline, Code, DeviceHub, Article, Email } from '@sparkpost/matchbox-icons';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import useStatus from 'hooks/useStatus';
+
+const StyledLink = styled(Box)<BoxProps & { $active?: boolean }>`
+  cursor: pointer;
+  overflow-wrap: anywhere;
+  a,
+  a:visited,
+  a:hover,
+  a:active {
+    color: inherit;
+    text-decoration: none;
+  }
+  &:hover {
+    ${({ $active }) => {
+      return css({
+        bg: $active ? 'blue.700' : 'gray.200',
+        color: $active ? 'white' : 'inherit',
+      });
+    }}
+  }
+  ${({ $active }) => {
+    return css({
+      bg: $active ? 'blue.700' : 'transparent',
+      color: $active ? 'white' : 'inherit',
+    });
+  }}
+`;
+
+const StatusColorMap = {
+  none: 'green.700',
+  minor: 'yellow.400',
+  major: 'brand.orange',
+  critical: 'red.700',
+};
+
 const DocsNavigation = () => {
-  return <div>Nav goes here</div>;
+  const { status } = useStatus();
+  const environment = getWindow();
+
+  return (
+    <Box width="260px" position="sticky" top="0">
+      <StyledLink $active={environment?.location?.pathname === '/docs'}>
+        <Link href="/docs" passHref>
+          <Box
+            as="a"
+            display="inline-block"
+            fontSize="200"
+            fontWeight="semibold"
+            py="200"
+            lineHeight="200"
+            px="500"
+          >
+            Help & API
+          </Box>
+        </Link>
+      </StyledLink>
+      <StyledLink px="500" py="200">
+        <Link href="/docs/getting-started/" passHref>
+          <a>
+            <Box as="span">Getting Started</Box>
+          </a>
+        </Link>
```

Comment:
So, one of the AC for the ticket was to have is

>  sidebar sources its data from the file structure of markdown files in content/docs
>    - Note: Momentum docs used a yaml file, but support doc navigation structure should be auto generated from content

I know the top level pages are all that show up currently in the support docs navigation, but considering we have sub files in the support docs directory, we should probably still aim to have the same functionality we have in momentum with expandable groups and stuff (http://localhost:3000/momentum)